be/src/vec/exec/file_scan_node.cpp

```diff
@@ -100,9 +102,33 @@ Status FileScanNode::start_scanners() {
     }
 
     _scanners_status.resize(_scan_ranges.size());
+    ThreadPoolToken* thread_token = _runtime_state->get_query_fragments_ctx()->get_token();
+    PriorityThreadPool* thread_pool = _runtime_state->exec_env()->scan_thread_pool();
```

Comment:
Not a good idea to use scan thread pool for external table scan.
But we can just leave it here. And later we should refactor this part.